src/api/lib/middlewares/pgCrud.js

```diff
@@ -0,0 +1,129 @@
+/* eslint func-names:0*/
+
+import koa from 'koa';
+import coBody from 'co-body';
+import koaRoute from 'koa-route';
+
+export default (queriesFactory, availableMethods = ['GET', 'POST', 'PUT', 'DELETE']) => {
+    const app = koa();
+    let queries;
+
+    app.use(function* (next) {
+        const clean = method => method.toLowerCase().trim();
+        const method = clean(this.method);
+        this.isAllowed = availableMethods.map(clean).indexOf(method) !== -1;
```

Comment:
I suggest to rename this attribute `methodIsAllowed` if it's accessible on all next middlewares.